content/en/getting-started/quickstart/index.md

```diff
@@ -0,0 +1,307 @@
+---
+title: "Quickstart"
+linkTitle: "Quickstart"
+weight: 30
+description: >
+  This quickstart gives an overview of how you can get a simple AWS application up and running on your local machine to understand local cloud development with LocalStack!
+cascade:
+  type: docs
+---
+
+In this quickstart guide, we'll walk you through the process of starting LocalStack on your local machine and deploying a [serverless image resizer application](https://github.com/localstack-samples/sample-serverless-image-resizer-s3-lambda) that utilizes several AWS services. This guide aims to help you understand how to use LocalStack for the development and testing of your AWS applications locally. It introduces you to the following key concepts:
+
+- Starting a LocalStack instance on your local machine.
+- Deploying an AWS serverless application infrastructure locally.
+- Running an automated integration test suite against local infrastructure.
+- Exploring the LocalStack Web Application to view deployed resources.
+- Destroying the local infrastructure you have provisioned.
+
```

Comment:
add a `#The example App` Section that briefly describes what we are building, ideally with screenshots or animated gif showing it in action.

note: I've noticed that we use the pikachu in the below screenshots. This is quite certainly a copy right violation, and we should only use images that are our own or are in public domain.